examples/01_typescript/src/app.tsx

```diff
@@ -28,7 +27,7 @@ const NAMES = [
 
 const nameAtom = atom(NAMES[0] as string);
 
-const pokemonAtom = trpc.pokemon.byId.atomWithQuery((get) => get(nameAtom));
+const pokemonAtom = trpc.pokemon.byId.atomWithQuery((get: Getter) => get(nameAtom));
```

Comment:
Is this change necessary? I strongly think `get` type should be inferred. `Getter` type isn't for ordinary use case.

Reply:
I'll check it today

Reply:
Unfortunately, I couldn't figure out how to do it without using the Getter type. I updated the jotai and @trpc packages and made the Prettier test pass successfully.

Reply:
This is not good. `atomWithQuery` is `any` type. So, it's basically broken.